### PR TITLE
Fix API example code of save_inference_model

### PR DESCRIPTION
**Pull request by zhhsplendid**
Fixed screenshot:

5c6297e1a13e464d42c8b884bd75a16b

test=develop

### CODE REVIEW

**Review by sneaxiy — DISMISSED** [review]
LGTM.

**Review by shanyi15 — DISMISSED** [review]

File: python/paddle/fluid/io.py [hunk 1]
```diff
@@ -913,13 +913,32 @@ def save_inference_model(dirname,
...
+
+            # Save inference model. Note we don't save label and loss in this example
+            fluid.io.save_inference_model(dirname=path,
+                                          feeded_var_names=['img'],
+                                          target_vars=[predict],
+                                          executor=exe)
 
             # In this exsample, the function will prune the default main program
-            # to make it suitable for infering the `predict_var`. The pruned
+            # to make it suitable for infering the `predict` var. The pruned
```
Comment (chengduoZH) [inline python/paddle/fluid/io.py:941]: `exsample `-> `example`

Comment (zhhsplendid) [reply python/paddle/fluid/io.py:941]: Done, also found the same typo in another function. Fixed both of them

**Review by chengduoZH — APPROVED** [review]
LGTM

**Review by shanyi15 — APPROVED** [review]

**Pull request merged**